package/batocera/emulationstation/batocera-es-system/es_systems.yml

```diff
@@ -719,9 +719,9 @@ scummvm:
   platform:   scummvm
   emulators:
     scummvm:
-      scummvm: { requireAnyOf: [BR2_PACKAGE_SCUMMVM]          }
```

Comment:
Keep the old one, if you build locally.

scummvm: { requireAnyOf: [BR2_PACKAGE_SCUMMVM, BR2_PACKAGE_HAS_SCUMMVM] }